### ISSUE TITLE
version mismatch for push-jobs-client-2.1.0?

### ISSUE THREAD

**Issue by gliptak** [chef/opscode-pushy-client#99]
On a server with https://packages.chef.io/stable/el/7/push-jobs-client-2.1.0-1.el7.x86_64.rpm installed, the pushy-client version shows:

```
# /opt/push-jobs-client/bin/pushy-client --version
Push Client: 2.0.2
```

Is this expected?

Thanks

**Comment by tduffield** [comment]
This should be fixed in the 2.1.1 release.

**Comment by gliptak** [comment]
With https://packages.chef.io/stable/el/7/:push-jobs-client-2.1.1-1.el7.x86_64.rpm
```
$ /opt/push-jobs-client/bin/pushy-client --version
Push Client: 2.1.1
```
Thanks
